File: src/main/java/com/kmarzecki/communicator/util/InternationalizationUtil.java
package com.kmarzecki.communicator.util;

import com.kmarzecki.communicator.model.Language;

public class InternationalizationUtil {
    private static Language language = Language.EN;

    public static String userExists(String user){
        return language == Language.PL
                ? "Użytkownik " + user + " już istnieje"
                : "User " + user + " exists";
    }

    public static String cannotFindRequestedUsers(Language language){
        return language == Language.PL
                ?  "Nie udało się naleźć żądanych użytkowników"
                : "Cannot find all requested users";
    }

    public static String cannotBeYourOwnFriend(Language language){
        return language == Language.PL
                ?  "Nie możesz być sam sobie przyjacielem :)"
                : "You cannot be your own friend :)";
    }

    public static String alreadyAFriendOrInProgress(Language language){
        return language == Language.PL
                ?  "Już go znasz / nie odpowiedział na zapytanie"
                : "Already a friend or in progress of becoming one";
    }

    public static String conversationNameIsNotUnique(Language language){
        return language == Language.PL
                ?  "Nazwa konwersacji nie jest unikalna"
                : "Conversation name is not unique";
    }
    public static String cannotRespondToSomeoneElsesRequest(Language language){
        return language == Language.PL
                ?  "Nie można odpowiedzieć na czyjeś zapytanie"
                : "Cannot respond to someone else's request";
    }
    public static String cannotFindRequestedRequest(Language language){
        return language == Language.PL
                ?  "Nie udało się naleźć żądanej operacji"
                : "Such request does not exist";
    }
}

File: src/main/kotlin/com/kmarzecki/trollo/util/InternationalizationUtil.kt
package com.kmarzecki.trollo.util

import com.kmarzecki.trollo.model.Language

object InternationalizationUtil {
    private val language = Language.EN
    fun userExists(user: String): String {
        return if (language == Language.PL) "Użytkownik $user już istnieje" else "User $user exists"
    }

    fun cannotFindRequestedUsers(language: Language): String {
        return if (language == Language.PL) "Nie udało się naleźć żądanych użytkowników" else "Cannot find all requested users"
    }

    fun cannotBeYourOwnFriend(language: Language): String {
        return if (language == Language.PL) "Nie możesz być sam sobie przyjacielem :)" else "You cannot be your own friend :)"
    }

    fun alreadyAFriendOrInProgress(language: Language): String {
        return if (language == Language.PL) "Już go znasz / nie odpowiedział na zapytanie" else "Already a friend or in progress of becoming one"
    }

    fun conversationNameIsNotUnique(language: Language): String {
        return if (language == Language.PL) "Nazwa konwersacji nie jest unikalna" else "Conversation name is not unique"
    }

    fun cannotRespondToSomeoneElsesRequest(language: Language): String {
        return if (language == Language.PL) "Nie można odpowiedzieć na czyjeś zapytanie" else "Cannot respond to someone else's request"
    }

    fun cannotFindRequestedRequest(language: Language): String {
        return if (language == Language.PL) "Nie udało się naleźć żądanej operacji" else "Such request does not exist"
    }
}